### PR TITLE
sysvar: add 12 sysvars to docs-cn

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)
sysvar: add a few sysvars to docs-cn

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.4 (TiDB 6.4 versions)
- [x] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [x] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -1061,6 +1069,14 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 - 默认值：`ON`
 - 这个变量用来设置是否开启 `LIST (COLUMNS) TABLE PARTITION` 特性。
 
+### `tidb_enable_local_txn`
```
Comment (TomShawn) [inline system-variables.md:1072]: v6.1 hidden

File: system-variables.md [hunk 2]
```diff
@@ -1741,6 +1766,23 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
 - 如果一条查询语句执行过程中使用的内存空间超过该阈值，会触发系统变量 [`tidb_mem_oom_action`](#tidb_mem_oom_action-从-v610-版本开始引入) 中指定的行为。
 - 在 v6.1.0 之前这个开关通过 TiDB 配置文件 (`mem-quota-query`) 进行配置，且作用域为 `SESSION`。升级到 v6.1.0 时会自动继承原有设置，作用域变更为 `SESSION | GLOBAL`。
 
+### `tidb_memory_debug_mode_alarm_ratio`
```
Comment (TomShawn) [inline system-variables.md:1769]: only v6.2

File: system-variables.md [hunk 3]
```diff
@@ -1741,6 +1766,23 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
...
+
+- 作用域：SESSION
+- 是否持久化到集群：否
+- 类型：浮点型
+- 默认值：`0`
+- 该变量用于 TiDB 内部测试，**不推荐修改该变量值**。表示在 TiDB memory debug 模式下，允许的内存统计误差值。
+
+### `tidb_memory_debug_mode_min_heap_inuse`
```
Comment (TomShawn) [inline system-variables.md:1777]: only v6.2

File: system-variables.md [hunk 4]
```diff
@@ -1033,6 +1033,14 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 - 默认值：`OFF`
 - 这个变量用来控制是否开启 General Plan Cache。
 
+### `tidb_enable_historical_stats`
```
Comment (TomShawn) [inline system-variables.md:1036]: only in v6.2

File: system-variables.md [hunk 5]
```diff
@@ -2660,6 +2712,23 @@ Query OK, 0 rows affected, 1 warning (0.00 sec)
 
 这个变量是 `transaction_isolation` 的别名。
 
+### `tx_read_ts`
```
Comment (TomShawn) [inline system-variables.md:2715]: v6.1 hidden

File: system-variables.md [hunk 6]
```diff
@@ -2660,6 +2712,23 @@ Query OK, 0 rows affected, 1 warning (0.00 sec)
...
+
+- 作用域：SESSION
+- 是否持续化到集群：否
+- 默认值：`0`
+- 该变量仅用于 TiDB 内部实现，**不推荐设置该变量**。
+- 在 Stale Read 场景下，该会话变量用于帮助记录 Stable Read ts 值。
+
+### `txn_scope`
```
Comment (TomShawn) [inline system-variables.md:2723]: v6.1 hidden

File: system-variables.md [hunk 7]
```diff
@@ -1046,7 +1054,15 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 - 默认值：`ON`
 - 这个变量用于控制是否开启 index merge 功能。
 
-### tidb_enable_legacy_instance_scope <span class="version-mark">从 v6.0.0 版本开始引入</span>
+### `tidb_enable_index_merge_join`
```
Comment (TomShawn) [inline system-variables.md:1057]: v6.1 hidden

File: system-variables.md [hunk 8]
```diff
@@ -2349,6 +2415,14 @@ Query OK, 0 rows affected, 1 warning (0.00 sec)
 - 范围：`[-1, 256]`
 - TiFlash 中 request 执行的最大并发度。默认值为 -1，表示该系统变量无效。0 表示由 TiFlash 系统自动设置该值。
 
+### `tidb_mpp_store_fail_ttl`
```
Comment (TomShawn) [inline system-variables.md:2418]: @littlefall

File: system-variables.md [hunk 9]
```diff
@@ -1826,7 +1892,7 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
 - 默认值：`OFF`
 - 这个变量用来设置优化器是否执行聚合函数下推到 Join，Projection 和 UnionAll 之前的优化操作。当查询中聚合操作执行很慢时，可以尝试设置该变量为 ON。
 
-### `tidb_opt_cartesian_bcj`
+### `tidb_opt_broadcast_cartesian_join`
```
Comment (TomShawn) [inline system-variables.md:1895]: @windtalker

File: system-variables.md [hunk 10]
```diff
@@ -1488,6 +1520,15 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
 - 如果将该变量值设为 `ON`，从 PD 获取的时间戳的操作会被省掉，这种情况下只保证因果一致性但不保证线性一致性。详情请参考 PingCAP 博文 [Async Commit 原理介绍](https://pingcap.com/zh/blog/async-commit-principle)。
 - 对于需要只保证因果一致性的场景，可将此变量设为 `ON` 以提升性能。
 
+### `tidb_hash_exchange_with_new_collation`
```
Comment (TomShawn) [inline system-variables.md:1523]: @windtalker

File: system-variables.md [hunk 11]
```diff
@@ -1147,6 +1171,14 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 - 默认值：0
 - 这个变量用于控制是否开启 Apply 算子并发，并发数由 `tidb_executor_concurrency` 变量控制。Apply 算子用来处理关联子查询且默认无并发，所以执行速度较慢。打开 Apply 并发开关可增加并发度，提高执行速度。目前默认关闭。
 
+### `tidb_enable_piplelined_window_function`
```
Comment (TomShawn) [inline system-variables.md:1174]: @XuHuaiyu

File: system-variables.md [hunk 12]
```diff
@@ -1749,6 +1807,14 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
 - TiDB 内存使用占总内存的比例超过一定阈值时会报警。该功能的详细介绍和使用方法可以参考 [`memory-usage-alarm-ratio`](/tidb-configuration-file.md#memory-usage-alarm-ratio-从-v409-版本开始引入)。
 - 该变量的初始值可通过 [`memory-usage-alarm-ratio`](/tidb-configuration-file.md#memory-usage-alarm-ratio-从-v409-版本开始引入) 进行配置。
 
+### `tidb_merge_join_concurrency`
```
Comment (TomShawn) [inline system-variables.md:1810]: @XuHuaiyu

File: system-variables.md [hunk 13]
```diff
@@ -2436,6 +2510,14 @@ Query OK, 0 rows affected, 1 warning (0.00 sec)
 - 单位：秒
 - 这个变量设置了 [statement summary tables](/statement-summary-tables.md) 的刷新时间。
 
+### `tidb_streamagg_concurrency`
```
Comment (TomShawn) [inline system-variables.md:2513]: @XuHuaiyu

Comment (XuHuaiyu) [reply system-variables.md:2513]: LGTM

File: system-variables.md (hunk 12 above)
Comment (XuHuaiyu) [reply system-variables.md:1810]: LGTM

File: system-variables.md (hunk 11 above)
Comment (XuHuaiyu) [reply system-variables.md:1174]: LGTM

File: system-variables.md (hunk 7 above)
Comment (XuHuaiyu) [reply system-variables.md:1057]: LGTM

**Review by XuHuaiyu — APPROVED** [review]

File: system-variables.md (hunk 4 above)
Comment (chrysan) [reply system-variables.md:1036]: 这个变量存在一段时间了，不止 v6.2 有

Comment (chrysan) [reply system-variables.md:1036]: LGTM

File: system-variables.md (hunk 10 above)
Comment (windtalker) [reply system-variables.md:1523]: LGTM

File: system-variables.md (hunk 9 above)
Comment (windtalker) [reply system-variables.md:1895]: LGTM

File: system-variables.md (hunk 2 above)
Comment (wshwsh12) [reply system-variables.md:1785]: LGTM

File: system-variables.md (hunk 3 above)
Comment (wshwsh12) [reply system-variables.md:1793]: LGTM

File: system-variables.md (hunk 6 above)
Comment (Yisaer) [reply system-variables.md:2753]: lgtm

File: system-variables.md (hunk 5 above)
Comment (Yisaer) [reply system-variables.md:2745]: lgtm

File: system-variables.md [hunk 14]
```diff
@@ -1046,7 +1054,15 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
...
 - 这个变量用于控制是否开启 index merge 功能。
 
-### tidb_enable_legacy_instance_scope <span class="version-mark">从 v6.0.0 版本开始引入</span>
+### `tidb_enable_index_merge_join`
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 类型：布尔型
+- 默认值：`false`
+- 表示是否启用 `IndexMergeJoin` 算子。该变量为 TiDB 内部变量，**不推荐使用**，否则可能会造成数据正确性问题。
```
Comment (TomShawn) [inline system-variables.md:1063]:
```suggestion
- 表示是否启用 `IndexMergeJoin` 算子。
- 该变量为 TiDB 内部变量，**不推荐使用**，否则可能会造成数据正确性问题。
```

File: system-variables.md [hunk 15]
```diff
@@ -1488,6 +1520,15 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
...
 - 对于需要只保证因果一致性的场景，可将此变量设为 `ON` 以提升性能。
 
+### `tidb_hash_exchange_with_new_collation`
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 类型：布尔型
+- 默认值：`true`
+- 该变量为 TiDB 内部变量，**不推荐设置该变量**。
+- 该值表示是否在开启 new collation 的集群里生成 MPP hash partition exchange 算子。`true` 表示生成此算子，`false`表示不生成。
```
Comment (TomShawn) [inline system-variables.md:1530]:
```suggestion
- 该值表示是否在开启 new collation 的集群里生成 MPP hash partition exchange 算子。`true` 表示生成此算子，`false`表示不生成。
- 该变量为 TiDB 内部变量，**不推荐设置该变量**。
```

File: system-variables.md [hunk 16]
```diff
@@ -1741,6 +1782,23 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
 - 如果一条查询语句执行过程中使用的内存空间超过该阈值，会触发系统变量 [`tidb_mem_oom_action`](#tidb_mem_oom_action-从-v610-版本开始引入) 中指定的行为。
 - 在 v6.1.0 之前这个开关通过 TiDB 配置文件 (`mem-quota-query`) 进行配置，且作用域为 `SESSION`。升级到 v6.1.0 时会自动继承原有设置，作用域变更为 `SESSION | GLOBAL`。
 
+### `tidb_memory_debug_mode_alarm_ratio`
+
+- 作用域：SESSION
+- 是否持久化到集群：否
+- 类型：浮点型
+- 默认值：`0`
+- 该变量用于 TiDB 内部测试，**不推荐修改该变量值**。表示在 TiDB memory debug 模式下，允许的内存统计误差值。
```
Comment (TomShawn) [inline system-variables.md:1791]:
```suggestion
- 该变量表示在 TiDB memory debug 模式下，允许的内存统计误差值。
- 该变量用于 TiDB 内部测试，**不推荐修改该变量值**。
```

File: system-variables.md [hunk 17]
```diff
@@ -2660,6 +2742,23 @@ Query OK, 0 rows affected, 1 warning (0.00 sec)
 
 这个变量是 `transaction_isolation` 的别名。
 
+### `tx_read_ts`
+
+- 作用域：SESSION
+- 是否持续化到集群：否
+- 默认值：`0`
+- 该变量仅用于 TiDB 内部实现，**不推荐设置该变量**。
+- 在 Stale Read 场景下，该会话变量用于帮助记录 Stable Read ts 值。
```
Comment (TomShawn) [inline system-variables.md:2751]:
```suggestion
- 在 Stale Read 场景下，该会话变量用于帮助记录 Stable Read ts 值。
- 该变量仅用于 TiDB 内部实现，**不推荐设置该变量**。
```

File: system-variables.md [hunk 18]
```diff
@@ -2660,6 +2742,23 @@ Query OK, 0 rows affected, 1 warning (0.00 sec)
...
+
+### `txn_scope`
+
+- 作用域：SESSION
+- 是否持续化到集群：否
+- 默认值：`global`
+- 可选值：`global` 和 `local`
+- 该变量仅用于 TiDB 内部实现，**不推荐设置该变量**。
+- 该变量用于设置当前会话下事务为全局事务还是局部事务。
```
Comment (TomShawn) [inline system-variables.md:2760]:
```suggestion
- 该变量用于设置当前会话下事务为全局事务还是局部事务。
- 该变量仅用于 TiDB 内部实现，**不推荐设置该变量**。
```

File: system-variables.md [hunk 19]
```diff
@@ -1749,6 +1809,14 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
 - TiDB 内存使用占总内存的比例超过一定阈值时会报警。该功能的详细介绍和使用方法可以参考 [`memory-usage-alarm-ratio`](/tidb-configuration-file.md#memory-usage-alarm-ratio-从-v409-版本开始引入)。
 - 该变量的初始值可通过 [`memory-usage-alarm-ratio`](/tidb-configuration-file.md#memory-usage-alarm-ratio-从-v409-版本开始引入) 进行配置。
 
+### `tidb_merge_join_concurrency`
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 默认值：`1`
+- 设置 `MergeJoin` 算子执行查询时的并发度。
+- 警告：**不推荐设置该变量，修改该变量值可能会造成数据正确性问题**。
```
Comment (Oreoxmt) [inline system-variables.md:1818]:
```suggestion
- **不推荐设置该变量**，修改该变量值可能会造成数据正确性问题。
```

File: system-variables.md [hunk 20]
```diff
@@ -2660,6 +2744,23 @@ Query OK, 0 rows affected, 1 warning (0.00 sec)
 
 这个变量是 `transaction_isolation` 的别名。
 
+### `tx_read_ts`
+
+- 作用域：SESSION
+- 是否持续化到集群：否
+- 默认值：`0`
+- 在 Stale Read 场景下，该会话变量用于帮助记录 Stable Read ts 值。
```
Comment (Oreoxmt) [inline system-variables.md:2752]:
```suggestion
- 在 Stale Read 场景下，该会话变量用于帮助记录 Stable Read TS 值。
```

File: system-variables.md [hunk 21]
```diff
@@ -2436,6 +2512,14 @@ Query OK, 0 rows affected, 1 warning (0.00 sec)
 - 单位：秒
 - 这个变量设置了 [statement summary tables](/statement-summary-tables.md) 的刷新时间。
 
+### `tidb_streamagg_concurrency`
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 默认值：`1`
+- 设置 `StreamAgg` 算子执行查询时的并发度。
+- 警告：**不推荐设置该变量，修改该变量值可能会造成数据正确性问题**。
```
Comment (Oreoxmt) [inline system-variables.md:2521]:
```suggestion
- **不推荐设置该变量**，修改该变量值可能会造成数据正确性问题。
```

File: system-variables.md [hunk 22]
```diff
@@ -2660,6 +2744,23 @@ Query OK, 0 rows affected, 1 warning (0.00 sec)
...
+- 在 Stale Read 场景下，该会话变量用于帮助记录 Stable Read ts 值。
+- 该变量仅用于 TiDB 内部实现，**不推荐设置该变量**。
+
+### `txn_scope`
+
+- 作用域：SESSION
+- 是否持续化到集群：否
+- 默认值：`global`
+- 可选值：`global` 和 `local`
+- 该变量用于设置当前会话下事务为全局事务还是局部事务。
```
Comment (Oreoxmt) [inline system-variables.md:2761]:
```suggestion
- 该变量用于设置当前会话下事务为全局事务（设为 `global`）还是局部事务（设为 `local`）。
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

File: system-variables.md [hunk 23]
```diff
@@ -1046,7 +1054,16 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 - 默认值：`ON`
 - 这个变量用于控制是否开启 index merge 功能。
 
-### tidb_enable_legacy_instance_scope <span class="version-mark">从 v6.0.0 版本开始引入</span>
+### `tidb_enable_index_merge_join`
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 类型：布尔型
+- 默认值：`false`
```
Comment (TomShawn) [inline system-variables.md:1062]:
```suggestion
- 默认值：`OFF`
```

File: system-variables.md (hunk 11 above)
Comment (TomShawn) [reply system-variables.md:1175]:
```suggestion
### `tidb_enable_pipelined_window_function`
```

File: system-variables.md [hunk 24]
```diff
@@ -1147,6 +1172,14 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 - 默认值：0
 - 这个变量用于控制是否开启 Apply 算子并发，并发数由 `tidb_executor_concurrency` 变量控制。Apply 算子用来处理关联子查询且默认无并发，所以执行速度较慢。打开 Apply 并发开关可增加并发度，提高执行速度。目前默认关闭。
 
+### `tidb_enable_piplelined_window_function`
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 类型：布尔型
+- 默认值：`true`
```
Comment (TomShawn) [inline system-variables.md:1180]:
```suggestion
- 默认值：`ON`
```

File: system-variables.md [hunk 25]
```diff
@@ -1488,6 +1521,15 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
 - 如果将该变量值设为 `ON`，从 PD 获取的时间戳的操作会被省掉，这种情况下只保证因果一致性但不保证线性一致性。详情请参考 PingCAP 博文 [Async Commit 原理介绍](https://pingcap.com/zh/blog/async-commit-principle)。
 - 对于需要只保证因果一致性的场景，可将此变量设为 `ON` 以提升性能。
 
+### `tidb_hash_exchange_with_new_collation`
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 类型：布尔型
+- 默认值：`true`
```
Comment (TomShawn) [inline system-variables.md:1529]:
```suggestion
- 默认值：`ON`
```

**Pull request merged**